src/generators.rs

```diff
@@ -308,4 +350,36 @@ mod tests {
         helper(16, 2);
         helper(16, 1);
     }
+
+    #[test]
+    fn add_gens_matches() {
```

Comment:
small nit: add gens matches what?
```suggestion
    fn resizing_small_gens_matches_creating_bigger_gens() {
```